bmorph/core/bmorph.py

```diff
@@ -0,0 +1,111 @@
+"""
+bmorph: modify a time series by removing elements of persistent differences
+        (aka bias correction)
+
+Persistent differences are inferred by comparing a 'truth' sample with a
+'training' sample. These differences are then used to correcta 'raw' sample
+that is presumed to have the same persistent differences as the 'training'
+sample. The resulting 'bmorph' sample should then be consistent with the
+'truth' sample.
+
+Here all samples are expected to be pandas dataframes and/or Series
+"""
+
+import scipy.stats
+
+
+def presrat(raw_x, raw_cdf, train_cdf, truth_cdf):
```

Comment:
I picked the name because it is a "known" algorithm implemented by others (e.g. David Pierce) with that name. I have clarified that in the doc string and added the references.